packages/babel-runtime-corejs3/helpers/dispose.js

```diff
@@ -1,17 +1,18 @@
-var _Object$create = require("@babel/runtime-corejs3/core-js/object/create");
-var _Promise = require("@babel/runtime-corejs3/core-js/promise");
+var _SuppressedError = require("core-js-pure/features/suppressed-error.js");
+var _Object$create = require("core-js-pure/features/object/create.js");
+var _Promise = require("core-js-pure/features/promise/index.js");
 function dispose_SuppressedError(suppressed, error) {
-  return dispose_SuppressedError = function dispose_SuppressedError(suppressed, error) {
+  return "undefined" != typeof _SuppressedError ? dispose_SuppressedError = _SuppressedError : (dispose_SuppressedError = function dispose_SuppressedError(suppressed, error) {
```

Comment:
We minify helpers before injecting polyfills and not after, however any users bundling our helpers in a client-side bundle will usually re-minify them (all bundlers minify the end bundle, and not just the individual chunks) and this will be very easily eliminated :)